airflow/operators/sensors.py

```diff
@@ -331,3 +331,57 @@ def poke(self, context):
         logging.info(
             'Checking if the time ({0}) has come'.format(self.target_time))
         return datetime.now().time() > self.target_time
+
+
+class HttpSensor(BaseSensorOperator):
+    """
+    Executes an HTTP get statement until the specified criteria is met.
+
+    :param conn_id: The connection to run the sensor against
+    :type conn_id: string
+    :param url: To pass, the url must respond with
+    """
+
+    @apply_defaults
+    def __init__(self,
+                 conn_id,
```

Comment:
`http_conn_id` because of how you can set default_args at your DAGs level, `conn_id` would conflict for default settings